code-challenges.js

```diff
@@ -31,24 +36,57 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilingPoint = 212
+const checkWaterTemp = (temp) => {
+    if (temp < boilingPoint) {
+        return `${temp} is below boiling point`
+    } else if (temp > boilingPoint){
+        return `${temp} is above boiling point`
+    } else if (temp === boilingPoint) {
+        return `${temp} is at boiling point`
+    } else {
+        return "Something is not right"
+    }
+}
+console.log(checkWaterTemp(temperature1))
+console.log(checkWaterTemp(temperature2))
+console.log(checkWaterTemp(temperature3))
+console.log(checkWaterTemp("apple"))
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// create variable for the combind arrays using .length named padresRecord
+// concatinate both arrays using .concat
+// consol.log padresRecord
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const padresRecord = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+console.log(`Here are ${padresRecord.length} Padres world series records`)
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+// create a new variable named mirrorImage to make currentCohort an array
+// use .spit to call the fuction to spit the strin
+// use .reverse to reverse the string 
+// use .join to recombinded the string in the reverse order
+
 
 const currentCohort = "Bravo 2023"
 // Expected output: "3202 ovarB"
 
+const mirrorImage = currentCohort.split('').reverse().join("")
+console.log(`Put that thang down flip it and reverse it "${mirrorImage}"`)
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
+// lastIndexOf for givenValue1
+// lastIndexOf for givenValue2
```

Comment:
Why use lastIndexOf? Your pseudocode should be able to describe why you intend to use that.